code/modules/awaymissions/corpse.dm

```diff
@@ -526,6 +526,10 @@
 	flavour_text = "Walk this mortal plain and terrorize all living adventurers who dare cross your path."
 	assignedrole = "Skeleton"
 
+/obj/effect/mob_spawn/human/skeleton/alive/equip(mob/living/carbon/human/H)
+	var/obj/item/implant/exile/Implant = new/obj/item/implant/exile(H)
```

Comment:
lowercase Implant here please

Reply:
both instances of capitalized implant if you do that